caddyhttp/fastcgi/fastcgi.go

```diff
@@ -286,6 +286,12 @@ func (h Handler) buildEnv(r *http.Request, rule Rule, fpath string) (map[string]
 	// Retrieve name of remote user that was set by some downstream middleware such as basicauth.
 	remoteUser, _ := r.Context().Value(httpserver.RemoteUserCtxKey).(string)
 
+	requestScheme := "http"
+
```

Comment:
Just a nit, but remove the empty line here. :)

Reply:
ok:)